pyzabbix/sender.py

```diff
@@ -131,6 +131,70 @@ def __repr__(self):
         return result
 
 
+class ZabbixActiveChecksResponse(object):
+    """The :class:`ZabbixActiveChecksResponse`
+    contains the parsed response from Zabbix.
+    """
+    def __init__(self):
+        self.checks = None
+
+    def __repr__(self):
+        """Represent detailed ZabbixActiveChecksResponse view."""
+        if self.checks is None:
+            return ''
+        else:
+            return "[%s]" % ", ".join([ch.__repr__() for ch in self.checks])
```

Comment:
It's better represent it as json. Which you doing in `ZabbixCheck.__repr__`.  I think it should be consisted.